lib/questions.rb

```diff
@@ -56,6 +68,7 @@ def make_numbers_negative(number)
 # even numbers come first
 # so [1, 2, 3, 4, 5, 6] becomes [[2, 4, 6], [1, 3, 5]]
 def separate_array_into_even_and_odd_numbers(array)
+  [array.select{|a| a.even?},array.select{|a| a.odd?}]
```

Comment:
Space missing after comma.<br>Space missing to the left of {.<br>Space between { and | missing.<br>Space missing inside }.<br>Pass `&:even?` as an argument to `select` instead of a block.<br>Pass `&:odd?` as an argument to `select` instead of a block.